pinot-core/src/main/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPruner.java

```diff
@@ -238,6 +237,66 @@ private boolean pruneRangePredicate(IndexSegment segment, RangePredicate rangePr
     return false;
   }
 
+  /**
+   * For IN predicate, segment will not be pruned if the size of values is greater than threshold
+   * Prune the segment based on:
+   * <ul>
+   *   <li>Column min/max value</li>
+   * </ul>
+   * Returns:
```

Comment:
Javadoc appears to be very verbose for this function and is taking up too many vertical lines. Can we keep it concise by getting rid of <ul>. Also I think "@return" is the standard way of specifying function return values (unless something has changed recently). I would suggest just having the following as javadoc:

/**
 * @return true if size of values is greater than threshold or if value is greater than min and less than max; 
 *  otherwise, return false.
 */

Reply:
done!